Framework/src/runnerUtils.cxx

```diff
@@ -91,6 +91,20 @@ std::string computeStringActivityField(framework::ServiceRegistryRef services, c
   return property;
 }
 
+std::string translateBeamType(const std::string& pdpBeamType)
+{
+  // convert the beam type received from pdp into the format we use in flp/ecs
+  if (pdpBeamType == "pp") {
+    return "PROTON-PROTON";
+  } else if (pdpBeamType == "PbPb") {
+    return "Pb-Pb";
+  } else if (pdpBeamType == "pPb") {
+    return "Pb-PROTON";
+  } else {
+    return "";
```

Comment:
Maybe add some warning, that we met value that we did not expect?

Reply:
yep

Reply:
done, I let you have a look